File: python/sprint1_nonfinals/e.py
def get_longest_word(line: str) -> str:
    # Здесь реализация вашего решения
    pass

def read_input() -> str:
    _ = input()
    line = input().strip()
    return line

def print_result(result: str) -> None:
    print(result)
    print(len(result))

print_result(get_longest_word(read_input()))

File: cpp/sprint1_nonfinals/e.cpp
#include <iostream>
#include <string>

using namespace std;

string GetLongestWord(const string &line) {
	// Здесь реализация вашего решения

}

void print(const string &result) {
	cout << result << endl << result.length() << endl;
}

int main() {
	int n;
	cin >> n;
	string line;
	getline(cin, line);
	getline(cin, line);
	print(GetLongestWord(line));
}
